.yarn/plugins/@yarnpkg/plugin-deduplicate.cjs

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+module.exports = {
+name: "@yarnpkg/plugin-deduplicate",
```

Comment:
@eps1lon I just realized that `yarn-plugin-deduplicate` is not licensed under any known licenses. Can you grant permission of usage for the Babel project?

Reply:
It's now licensed under MIT. Thanks for the notice :+1: